cmd/fuseml_core/main.go

```diff
@@ -119,13 +119,66 @@ func main() {
 				}
 				u.Host = net.JoinHostPort(h, *grpcPortF)
 			} else if u.Port() == "" {
-				u.Host = net.JoinHostPort(u.Host, ":8080")
+				u.Host = net.JoinHostPort(u.Host, "8080")
+			}
+			handleGRPCServer(ctx, u, runnableEndpoints, &wg, errc, logger, *dbgF)
+		}
+
+	case "prod":
+		{
+			addr := "http://0.0.0.0"
```

Comment:
Apart from this line, the code below is literally the same code.
Why not extracting this into a function/method which takes the address as argument, and be done with this?

Reply:
This code is auto-generated and we might want to improve it later. For now I prefer to leave it like this as this is not the intent of this PR.